Misha/Files/pdf_converter/pdf_converter_tests.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+from pdf_converter_main import read_from_file, change_format_to_pdf
+
+
+class PDFConverterTest(unittest.TestCase):
```

Comment:
Please add more tests
1) When big single line converted to pdf
2) When second page is used

Reply:
Also use separate folder (test-resource) for test files